scripts/build_platform_config.py

```diff
@@ -286,7 +286,7 @@ def codeOutDevicePin(device, pin, definition_name):
   codeOut("#define FLASH_SAVED_CODE_START            "+str(flash_saved_code_start))
   codeOut("#define FLASH_SAVED_CODE_LENGTH           "+str(int(flash_page_size*flash_saved_code_pages)))
   codeOut("#define FLASH_MAGIC_LOCATION              (FLASH_SAVED_CODE_START + FLASH_SAVED_CODE_LENGTH - 4)")
-  codeOut("#define FLASH_MAGIC 0xDEADBEEF")
+  codeOut("#define FLASH_MAGIC 0xBADBEEF8")
```

Comment:
I changed the flash magic 'cause I've seen problem when upgrading older boards, if this is a problem, I can revert it.